src/components/side-panel/property-inputs/Checkbox.tsx

```diff
@@ -0,0 +1,40 @@
+import { VoidFunctionComponent } from "react";
+
+interface Props {
+  property: any;
+  sectionName: string;
+  updateProperty: (sectionName: string, property: any, value: any) => void;
+}
+
+const Checkbox: VoidFunctionComponent<Props> = ({
+  property,
+  sectionName,
+  updateProperty,
+}) => {
+  const onCheckboxChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value: property, checked: value } = e.target;
+    const sectionName = e.target.id.split("-")[0];
```

Comment:
sectionName is derived from props, so we could use it directly from props instead of deriving it